Source/NSubstitute/Properties/AssemblyInfo.cs

```diff
@@ -23,4 +23,3 @@
 
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("f1571463-8354-493c-b67c-cd9cec9adf78")]
-
```

Comment:
Sorry - my VS butchered the Assembly info and this was a copy paste from the raw master.